src/main/java/io/ebeaninternal/api/TransactionEvent.java

```diff
@@ -29,16 +30,21 @@
    */
   private final transient boolean local;
 
+  private final long startMillis;
+
   private TransactionEventTable eventTables;
 
-  private transient TransactionEventBeans eventBeans;
+  private transient List<PersistRequestBean<?>> listenerNotify;
```

Comment:
listenerNotify ... only hold requests that MUST be processed post commit.  Don't put L2 cache requests in here.